### PR TITLE
[70] Implement Admin's job page

### PR DESCRIPTION
**Pull request by priasdevo**
## What did you do

- image
- Move job-list page to components add new components (Admin page)
- refactor job/list structure
- add chips/grey out to jobCard
- connect with Job/reports

## Demo

- https://dev.modela.miello.dev/job ( use admin account )

### CODE REVIEW

File: apps/web/src/modules/job/list/components/ActorCastingPage/hooks/useJobListData/index.ts
Comment (PKhing) [inline apps/web/src/modules/job/list/components/ActorCastingPage/hooks/useJobListData/index.ts:13]:
```suggestion
} from 'modules/job/list/pages/types'
```

File: apps/web/src/modules/job/list/components/ActorCastingPage/hooks/useJobListData/index.ts
Comment (PKhing) [inline apps/web/src/modules/job/list/components/ActorCastingPage/hooks/useJobListData/index.ts:108]: useMediaQuery?

File: apps/web/src/modules/job/list/components/AdminPage/hooks/useJobListData/index.ts
Comment (PKhing) [inline apps/web/src/modules/job/list/components/AdminPage/hooks/useJobListData/index.ts:13]:
```suggestion
} from 'modules/job/list/pages/types'
```

File: apps/web/src/modules/job/list/components/AdminPage/index.tsx
Comment (PKhing) [inline apps/web/src/modules/job/list/components/AdminPage/index.tsx:50]:
```suggestion
      <PlaceFill/>
```

File: apps/web/src/modules/job/list/components/AdminPage/index.tsx
Comment (PKhing) [inline apps/web/src/modules/job/list/components/AdminPage/index.tsx:116]: remove this since casting is unable to access this page

**Review by PKhing — CHANGES_REQUESTED** [review]
Both AdminPage and ActorCastingPage should be in the pages folder (since it is a page 😂 )

**Comment by PKhing** [comment]
check or uncheck "งานที่ได้รับการแจ้งปัญหา" has no different now. I think this might be from backend since the request is correct (or is it expected behavior @SuitedIceZ )

File: apps/web/src/modules/job/list/components/FilterContainer/index.tsx
Comment (miello) [inline apps/web/src/modules/job/list/components/FilterContainer/index.tsx:64]:
```suggestion
                  setState((prev) => ({ ...prev, cancleCheck: event.target.checked }))
```

File: apps/web/src/modules/job/list/pages/types.ts
Comment (miello) [inline apps/web/src/modules/job/list/pages/types.ts:18]: Please change all object key and types which related to this also
```suggestion
  cancelCheck: boolean
```

File: apps/web/src/modules/job/list/pages/types.ts
Comment (miello) [inline apps/web/src/modules/job/list/pages/types.ts:52]: Please change all object key and types which related to this also
```suggestion
  cancelCheck: false,
```

File: apps/web/src/modules/job/list/components/FilterContainer/index.tsx
Comment (miello) [inline apps/web/src/modules/job/list/components/FilterContainer/index.tsx:49]:
```suggestion
                  setState((prev) => ({ ...prev, reportCheck: event.target.checked }))
```

File: apps/web/src/modules/job/list/components/AdminPage/hooks/useJobListData/index.ts
Comment (miello) [inline apps/web/src/modules/job/list/components/AdminPage/hooks/useJobListData/index.ts:115]: Consider removing `console.log`

**Review by miello — CHANGES_REQUESTED** [review]

File: apps/web/src/modules/job/list/pages/types.ts
Comment (PKhing) [reply apps/web/src/modules/job/list/pages/types.ts:18]: ตาโคตรดี :joy:

**Comment by priasdevo** [comment]
> check or uncheck "งานที่ได้รับการแจ้งปัญหา" has no different now. I think this might be from backend since the request is correct (or is it expected behavior @SuitedIceZ )

เหมือนว่า database ของ dev อันที่ open อยู่ทุกอันโดน report ( เปิด close ด้วยหรือเช็คกับ beta ไม่มีปัญหา)

File: apps/web/src/modules/job/list/pages/ActorCastingPage/index.tsx
Comment (miello) [inline apps/web/src/modules/job/list/pages/ActorCastingPage/index.tsx:1]:
```suggestion
import { UserType } from '@modela/dtos'
```

File: apps/web/src/modules/job/list/pages/index.tsx
Comment (miello) [inline apps/web/src/modules/job/list/pages/index.tsx:1]:
```suggestion
import { UserType } from '@modela/dtos'
```

File: apps/web/src/modules/job/list/pages/AdminPage/index.tsx
Comment (miello) [inline apps/web/src/modules/job/list/pages/AdminPage/index.tsx:1]:
```suggestion
import { UserType } from '@modela/dtos'
```

**Review by miello — CHANGES_REQUESTED** [review]
That's all from me

**Review by miello — APPROVED** [review]
Please wait for review from @PKhing also

**Review by PKhing — APPROVED** [review]

**Pull request merged**
